roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Install EPEL repo for RedHat 7
+  yum:
+    name:
+      - "{{ epel_repo_7 }}"
+    state: present
+  become: yes
+  when: ansible_distribution_major_version == "7"
+
+- name: Install PG repo for RedHat 7
+  yum:
+    name:
+      - "{{ pg_rpm_repo_7 }}"
+    state: present
+  become: yes
+  when:
+    - ansible_distribution_major_version == "7"
+    - pg_type == 'PG'
+
+- name: Download EPEL GPG key for RedHat 8
+  rpm_key:
+    state: present
+    key: "{{ epel_gpg_key_8 }}"
+  when:
+    - ansible_distribution_major_version == "8"
+  become: yes
+
+- name: Download EDB GPG key for RedhHat 8
+  rpm_key:
+    state: present
+    key: "{{ edb_gpg_key_8 }}"
+  when:
+    - ansible_distribution_major_version == "8"
+  become: yes
+
+- name: Download PGDG GPG key for CentOS8/RHEL8
+  rpm_key:
+    state: present
+    key: "{{ pg_gpg_key_8 }}"
+  when:
+    - ansible_distribution_major_version == "8"
+  become: yes
+
+- name: Install EPEL for RedHat 8
+  dnf:
+    name:
+      - "{{ epel_repo_8 }}"
+    state: present
+  when:
+    - ansible_distribution_major_version == "8"
+  become: yes
+
+- name: Install PG repo for RedHat 8
+  yum:
```

Comment:
We should use the `dnf` module in that case.

Reply:
Fixed. I made some more changes in PG_RedHat_setuprepos.yml. Can you please review the new change?

Reply:
Sounds good.